### ISSUE TITLE
Should not be forced to use `public` directory

### ISSUE THREAD

**Issue by ming-codes** [bguiz/ember-sprite#1]
I don't think ember-sprite should force users to use `public` directory for sprites. As per ember-cli's guide on folder layout,

> `public/` This folder will be copied verbatim into the root of your built application. Use this for **assets that don’t have a build step**, such as images or fonts.

Sprite sheet generating is a build step, therefore it does not belong in `public`

**Comment by Dhaulagiri** [comment]
I found that this was becoming a problem for our app in a roundabout way.  Our builds were taking an inordinate amount of time in the asset rewrite step (~90 seconds).  After some debugging I realized it was because all our sprite icons (~500) were being fingerprinted and then each one was considered when rewriting values back into our app/vendor .js/.css files.  Each one *only* added 70ms but given the sheer number of icons this really added up.  Ideally these wouldn't even be part of the build output as @ming-codes says, but there doesn't seem to be a good way to avoid it.  

I was able to get around this by not fingerprinting these files at all in the `ember-cli-build.js` file:

```fingerprint: {
      exclude: ['images/sprite'],
    },```
